documentation/airport-terrain-generation.md

```diff
@@ -1,190 +1,185 @@
-Initial setup
-=============
+# Terrain Generation
 
-Install QGIS from http://www.qgis.org/en/site/forusers/download.html.
+## Introduction
+###### This guide will show you how to generate a terrain file for any airport in openScope. This is mainly done using QGIS, a free and open source Geographic Information System. The process can be quite tedious and often frustrating, so if you get stuck somewhere, you can talk to us on [Slack](http://slack.openscope.co/), where we will be happy to help you!
 
-Create a directory for working in, for this documentation it will be
-referred to as ~/projects/openscope-terrain/ but it can be any
-directory on your computer.
+## Initial setup
 
-Menu items are written like this:
-Project->Save
+#### 1. Install QGIS version 2.18 from the [QGIS website](http://www.qgis.org/en/site/forusers/download.html)
+*Note: QGIS version 3 is not compatible with the Zonal Statistics plugin.*
 
-The above means click the Project Menu, then click the Save item in
-it.
+#### 2. Install Zonal Statistics plugin
+* Open QGIS and navigate to `Plugins->Manage and install plugins...`
+* Search for `Zonal statistics plugin` and enable plugin
 
-One plugin needs to be enabled, the following instructions
-demonstrate the interactions for a dialog box which are specified
-indented under the menu item, like this:
+#### 3. Create a folder to dump all your files into.
+* The folder should be named after the airport's ICAO code, e.g. `~/KSEA/`
+*Note: Throughout this guide it will be referred to as `~/terrain/`*
 
-Plugins->Manage and install plugins...
-  Search for 'Zonal statistics plugin'
-  Ensure the plugin is checked in the middle pane
-  Click Close
+#### 4. Obtain elevation data
+* Go to [this site](http://viewfinderpanoramas.org/Coverage%20map%20viewfinderpanoramas_org3.htm) and download all the tiles which cover your airport and its surrounding area
+* Extract the zip files into a new folder `~/terrain/DEM/`
 
+#### 5. Obtain airspace coordinates
+* Copy all the coordinates within the airspace key of your airport file
+* Convert the coordinates to the right format using [this web app](https://jakcharvat.github.io/polygonConverter/) *(Thanks @jakcharvat for this wonderful tool!)*
```

Comment:
Maybe it should be mentioned that the airspace coordinates have to be in degree-only format to use them with the web app. (Eventhough the web app has code for converting deg min sec to deg, it didn't work for me)

Reply:
It can be in decimal degrees. If it's actually not taking that, it would be a bug. Can you reverify and if it's definitely broken, open a new issue.

Reply:
The decimal degree format works: f.e. [41.471, 27.19383333333]
Also the degree minute second format works: f.e. ["N40d59m78", "E27d11m65"]
But it can't take degree with decimal minutes: f.e. ["N41d28.26", "E27d11.63"]

Reply:
@jakcharvat

Reply:
Ok, I'll look into that. It's probably because I was working with a degree-minute-second-decimal format ["N50d35m48.47", "E013d47m59.46"] so I'll look for a way to work with a degree-minute-decimal format as well

Reply:
Done. Pushed an update to the app to support the format @Marcel510 . Lemme know if there's anything else broken please